package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "browser-extension",
+    "version": "1.0.0",
```

Comment:
Current version is `1.2`, but I don't think it matters in this context. I can see us forgetting to keep this value up-to-date, since it has no effect on the publishing of the extension.